src/python/turicreate/test/test_audio_functionality.py

```diff
@@ -28,7 +28,7 @@
 from turicreate.toolkits._internal_utils import _mac_ver
 
 # TODO: Remove for 6.0
-IS_PRE_6_0_RC = False
+IS_PRE_6_0_RC = True
```

Comment:
This no longer seems correct; `IS_PRE_6_0_RC` should now be false, since we are at 6.0.0rc0. Right?

Reply:
Yeah I was wrong. But whatI realized now is that we don't need the condition. I removed the test_sframe_sequence_iterator.py as it was testing old AC implementation. (as @nickjong mentioned above) And audio_functionality does not need it anymore after merging master. I removed it from one place in style transfer that needs to be checked in a machine with linux gpu!